example_configs/lldap-ha-auth.sh

```diff
@@ -66,5 +66,26 @@ fi
 
 DISPLAY_NAME=$(jq -r .displayName <<< $USER_JSON)
 
+IS_ADMIN=false
+if [[ ! -z "$3" ]] && jq -e '.groups|map(.displayName)|index("'"$3"'")' <<< "$USER_JSON" > /dev/null 2>&1; then
```

Comment:
I'm not going to pretend I understand this. I assume you tested it?

Reply:
i tested the case where a regular user with the local group and an admin user without the local group logged into homeassistant. the behaviour is correct.